README.md

```diff
@@ -345,6 +345,23 @@ would give the following resolutions:
 |b        |/scope2/scope3/foo.mjs |/b-3.mjs      |
 |c        |/scope2/scope3/foo.mjs |/c-1.mjs      |
 
+#### `<base>` element
+
+When `<base>` element is present in the document, all URLs and URL-like specifiers in the import map are converted to absolute URLs using the `href` from `<base>`.
+
+The right-hand side of the mapping should be absolute URLs starting with `./`. "Bare" relative URLs like `node_modules/moment/src/moment.js` will not work for reasons stated in [[Bare specifiers for JavaScript modules]](#bare-specifiers-for-javascript-modules).
```

Comment:
I don't think this paragraph is very related to this section. I think it is covered earlier in https://github.com/WICG/import-maps#bare-specifiers-for-javascript-modules. So would it be OK to just remove it?

Reply:
Hm, I thought it was worthwhile repeating here since script src will support both:
```
<script src="vue.runtime.esm.js">
<script src="./vue.runtime.esm.js">
```
whereas import map will only support the latter. But I agree that it can be considered redundant. I'll remove it.